lib/CLI.js

```diff
@@ -89,19 +89,19 @@ Example:
         };
       } else {
         this._processExit = (exitStatus) => {
-          logger.verbose(`Exiting Dredd process with status '${exitStatus}'.`);
+          logger.debug(`Exiting Dredd process with status '${exitStatus}'.`);
```

Comment:
This line is duplicated and since it contains a log message perhaps it would make sense to use the same string to prevent any potential for typos in messages only if Dredd is used with certain options.

Perhaps something like the following closure and using some form of composition to use it with the other two closures.

```js
const logExit = exitStatus => logger.debug(`Exiting Dredd process with status '${exitStatus}'.`);
```

Outside the scope of this PR but just a thought while reading.

Reply:
```js
// This thing-a-ma-bob here is only for purpose of testing
// It's basically a dependency injection for the process.exit function
```

If I'm ever touching that code, I'll do that in order to find and implement less crazy way how to test how the program exits.

A closure could be an improvement, but I'd rather see the code to go away.

Reply:
Tried to improve this in 67b6b2e46ae4e272ef6011fbf0daa9032b42d8a3

Reply:
This is completely out of the scope of these changes but a potentially useless idea I wanted to share.

I was also thinking / wondering how localisation is done for Node CLI apps (guessing that no one is tackling that). I've dealt with internationalisation / localisation in the past in other languages and environments and you usually end up one with language specific file that declares all of the human readable messages which might be nice to separate them from the logic. Makes it easier for people to change and potentially easier to spell check them in CI.

There's a nice example for iOS I wrote at https://github.com/cocodelabs/palaver-l10n/ (even has Czech locale and linting of the language files). In a past life I also wrote django-request which has localisation (https://github.com/django-request/django-request/tree/master/request/locale).

In this case we could have a language or similiar file that could contain all of the output messages, it might be a nice separation.

```js
// language-en.js

const EXIT_DEBUG_MESSAGE = exitStatus => `Exiting Dredd process with status '${exitStatus}'.`;
```


```js
// some-file.js

const { EXIT_DEBUG_MESSAGE } = require('./locale.js');

console.log(EXIT_DEBUG_MESSAGE(1));
```

Reply:
Seems nice! I also worked on some localized apps before, mostly using [Babel](https://pythonhosted.org/Flask-Babel/), which leverages gettext. Using constants like this could be a nice poor mans gettext. Would you file it as an idea to issues?

Reply:
I didn't nessecerily mean supporting localisation. I was thinking of it as a way to expose all the human readable output in a single source file instead of the user facing strings skattered across the code base.

Reply:
Oh, hmm. I don't think that's very useful for application logging. It seems to me it would rather obscure it for the reader. Perhaps for the output of the reporters, IDK. I don't see that much value in having the strings at one place if we don't do localisation. It's easier to read and edit it in place, within the context of the code which triggers it.